exec/cnex/test_string_support.c

```diff
@@ -78,6 +79,20 @@ int main()
     int64_t last = string_findLastNotOf(s3, ts);
     TString *r = string_subString(s3, first, last - first);
     assert(string_compareString(r, string_createCString("This is String#3's Data.\t\r\n")) == 0);
+    // Test Find operations
+    assert(string_findCharRev(foobar, '.') == 3);
+    assert(string_findCharRev(s2, ' ') == 19);
+    assert(string_findCharRev(foobar, '/') == NPOS);
+
+    size_t lastsep = string_findCharRev(path, '/');
+    TString *fname = NULL;
+    if (lastsep != NPOS) {
```

Comment:
This isn't a suitable way to write test code. You *know* what the return value is supposed to be, so check it directly with an assert test. Don't put conditional logic in tests like this.

Reply:
Actually, that little bit of code shouldn't even really be there.  There are tests for the rest of these calls already.  So, I'll remove it.